CMakeLists.txt

```diff
@@ -32,12 +34,13 @@ add_executable(enftun
   src/tun.c
 )
 
-target_link_libraries(enftun
-  PRIVATE
+target_link_libraries(enftun PRIVATE
   LibConfig::LibConfig
   LibUV::LibUV
   OpenSSL::SSL
-)
+  xtt::xtt
+  sodium
```

Comment:
This may go away if we decide to create an `xtt_handshake(...)` function, but currently there are some calls to libsodium functions `crypto_hash_256_(init, update, final)` in order to generate the gid from the gpk -- this could be a potential function in the XTT library.

Reply:
Gotcha.